base/js/admin.js

```diff
@@ -615,19 +616,22 @@
                     var $in = $(input);
 
                     if(typeof pos !== 'undefined') {
-                        var newName = $in.data('original-name');
+                        var newName = $in.attr('data-original-name');
```

Comment:
I'm confused. I thought the jQuery `data()` method effectively created an attribute with the given name and prefixed with `data-`. I'll have to go through the code to see what the difference is there.